learn-accessibility-by-building-a-website-for-people-with-disabilities/index550.html

```diff
@@ -0,0 +1,137 @@
+
+      <main id="main-content" class="main-content">
+        <h1 class="top-heading">Take our survey!</h1>
+        <img alt="One of our students coding a Tic Tac Toe game" src="assets/code-coder-coding-34649.jpg" class="image">
+        <form class="form">
+
+          <div class="form-group">
+            <fieldset>
+              <legend class="field-label">
+                Where did you find out about CodeHouse?
+              </legend>
+              <input id="google" name="where" type="radio"> <label for="google">Google search</label>
+              <input id="friend" name="where" type="radio"> <label for="friend">Word of mouth</label>
+              <input id="other1" name="where" type="radio"> <label id="other-label" for="other1">Other (please specify)</label>
+              <input aria-labelledby="other-label" id="other2" type="text" class="form-control other">
+            </fieldset>
+          </div>
+
+          <div class="form-group">
+            <label for="how-many" class="field-label">
+              How many of our events have you attended in the past?
+            </label>
+            <input type="number" id="how-many" class="form-control">
+          </div>
+
+          <fieldset>
+            <legend>
+              Start, stop, keep
+            </legend>
+
+            <div class="form-group">
+              <label for="start" class="field-label">
+                What’s one thing we should <em>start</em> doing?
+              </label>
+              <input type="text" id="start" class="form-control">
+            </div>
+
+            <div class="form-group">
+              <label for="stop" class="field-label">
+                What’s one thing we should <em>stop</em> doing?
+              </label>
+              <input type="text" id="stop" class="form-control">
+            </div>
+
+            <div class="form-group">
+              <label for="keep" class="field-label">
+                What’s one thing we should <em>keep</em> doing?
+              </label>
+              <input type="text" id="keep" class="form-control">
+            </div>
+          </fieldset>
+
+          <div class="form-group">
+            <label for="comments" class="field-label">
+              Any other comments?
+            </label>
+            <textarea class="form-control" id="comments"></textarea>
+          </div>
+
+          <div class="form-group">
+            <label for="email" class="field-label">Enter your email address to receive our newsletter:</label>
+            <input type="text" class="form-control" id="email" placeholder="name@example.com">
+          </div>
+
+          <button type="submit" class="btn">
+            Submit
+          </button>
+        </form>
+        <img alt="" src="assets/divider.png" class="divider">
+        <a href="#">Learn more about our Coffee and Code events.</a>
+      </main>
+      <!--
+        Great! Now we just have a couple more stylistic things to fix in `index.css`.
```

Comment:
Not sure exactly how the new UI will work, but we're assuming that the file the student will edit will be opened and focused on automatically. I think it's safe to remove this lesson.